### PR TITLE
Integrated recording transcript into word2vec model

### PR DESCRIPTION
**Pull request by denwychan**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -123,7 +150,7 @@
                 df = df[df['Ingredients_Length']>=min_number_ingredients]
 
             #using search function to find no.1 matching recipe
-            top_recipes = shortlist_recipes(df, output, df.index)
+            top_recipes = shortlist_recipes(df, transcript, df.index)
```
Comment (LukeG28) [inline app.py:153]: Can you feed the shortlist from the vector model into the shortlist recipes function?

File: app.py [hunk 2]
```diff
@@ -123,7 +150,7 @@
...
             no_1 = top_recipes[0][0]
             title = df.loc[no_1, 'Title']
```
Comment (LukeG28) [inline app.py:155]: Can you incorporate latest from master into this, the code has changed (simplified) quite a bit

**Review by LukeG28 — CHANGES_REQUESTED** [review]
I've added a couple of comments near bottom of app file, lemme know if anything unclear

**Comment by denwychan** [comment]
Closing pull request to work on changes

**Pull request closed without merging**